src/types/Document.ts

```diff
@@ -48,6 +48,11 @@ export class Document {
    * Document Version (generated by Elasticsearch)
    */
   _version?: number;
+  /**
+   * For upsert: if true, a new document was created,
+   * otherwise the document existed and was updated
```

Comment:
```suggestion
   * If true, a new document was created,
   * otherwise the document existed and was updated
   * @see https://docs.kuzzle.io/sdk/js/7/controllers/document/upsert/
```